sgx/ias-types/src/quote.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! IAS Quote Structure
```

Comment:
This is the "special" version of `sgx_quote_t` that's returned by IAS (it does not contain the `signature_len` unsigned or variable-length `signature` fields) but not actually provided anywhere in the SGX headers. We skip the byte representation, as it's never used during FFI, in favor of parsing it directly into the rusty types that sit above the FFI types.

This was handled in the earlier attest crates by special-casing these "short" `sgx_quote_t` variants, rather than creating a whole new type, which is the approach here.

Reply:
Might be worth including part of this explanation in the comment.